docs/testing/component-tests.md

```diff
@@ -21,14 +21,14 @@ This obviously varies between components, but a few easy things to start out wit
 Like its child components?
 
 ```javascript
-import AnExpectedChildComponent from 'an-expected-child-component';
-import { shallow } from 'enzyme';
+import Dialog from '@automattic/components';
```

Comment:
Would importing `Dialog` be important for this test, assuming `MyComponent` already imports it and we just look it up by role?

Reply:
Good eye, I think it can be removed 👍